main.js

```diff
@@ -645,6 +648,10 @@ function updateIniFromForm(ini) {
             ini.reset.s_key = "0";
         }
     }
+    var fun = parseInt(document.getElementById("ini-fun").value);
+    if(fun){
```

Comment:
Sorry, yes! Will do tomorrow. If you don't feel like changing it. I left the "allow edits from maintainers" checkbox on